mlir/utils/jenkins/Dockerfile

```diff
@@ -125,3 +125,16 @@ RUN wget https://raw.githubusercontent.com/ROCmSoftwarePlatform/MIOpen/develop/r
     && cmake -P install_deps.cmake --prefix /usr/local
 RUN echo $(git ls-remote https://github.com/ROCmSoftwarePlatform/MIOpen HEAD) > miopen-deps-commit-hash
 WORKDIR /
+# --------------------- Section 4: MIGraphX dependencies ---------------
+WORKDIR /MIGraphXDeps
+RUN pip3 install setuptools wheel
+# Install MIGraphX python dependencies
+RUN pip3 install https://github.com/RadeonOpenCompute/rbuild/archive/master.tar.gz
+RUN pip3 install onnx==1.8.1 numpy==1.21.6 typing==3.7.4 pytest==6.0.1 packaging==16.8
+RUN pip3 install protobuf==3.20.0
+# Get the requirement files from the commit be70702dc80d59126d6b4011d6cc7913193db566
```

Comment:
First, let's parametrize this like we do MIOpen.

Second, note that we already `pip install` some of these elsewhere in the Dockerfile.

Third, I don't like having tight version bounds like this somewhere where we won't think to touch them - can we drop the `==` bits? Unless we already had a bunch of those.

Reply:
Parameterize which part?
(If you can point me to that in this file)

I can have a go which parts we can drop from pip installs...

Reply:
It's not that I'm opposed to pinning a version, it's that I don't want us to not bump the version because MIGraphX changed it and we didn't notice

Reply:
Parametrize the MIGraphX commit. And I take that comment back because we just use MIOpen's develop at the time of Docker build

Reply:
Ack about commit.

Re: pinned versions

Not sure by not pinning versions, we can avoid having to bump the versions. I mean if it does break with implicit versions, we have to bump anyway.

I ll try and see whats really necessary here

Reply:
Eventually if there's similar amount of tests from MIGraphx we'd want to follow the convention from MIOpen and remove that pinned version. We should theoretically always need the TOT in each docker builds and manually bumped commit hash is a pain to manage by hand. I'm good with what you have now after the parameterized change.